dev_guide/deployments.adoc

```diff
@@ -295,8 +318,8 @@ triggers:
 [[image-change-trigger]]
 === Image Change Trigger
 
-The `*ImageChange*` trigger results in a new deployment whenever the value of
-an image stream tag changes, either by a build or because it was imported.
+The ImageChange trigger results in a new replication controller whenever the value of an
+image stream tag changes.
```

Comment:
the content of an image stream tag changes (iow. a new version of image is pushed)